nilearn/image/image.py

```diff
@@ -842,7 +862,45 @@ def threshold_img(img, threshold, mask_img=None, copy=True):
                                            percentile_func=scoreatpercentile,
                                            name='threshold')
 
-    img_data[np.abs(img_data) < cutoff_threshold] = 0.
+    # Apply threshold
```

Comment:
I think the function is a bit long and could be simplified:
First, I think we don't need to check if the threshold is `None` to raise a `ValueError` (see above), as `check_threshold` already does (almost) this (raises a `TypeError`).

Reply:
Makes sense. Fixed!